[2024-12-03T13:00:38.029+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-03T13:00:38.065+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_process_dag.load_transaction_data manual__2024-12-03T13:00:33.897752+00:00 [queued]>
[2024-12-03T13:00:38.077+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_process_dag.load_transaction_data manual__2024-12-03T13:00:33.897752+00:00 [queued]>
[2024-12-03T13:00:38.078+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-12-03T13:00:38.092+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_transaction_data> on 2024-12-03 13:00:33.897752+00:00
[2024-12-03T13:00:38.098+0000] {standard_task_runner.py:63} INFO - Started process 4338 to run task
[2024-12-03T13:00:38.101+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_process_dag', 'load_transaction_data', 'manual__2024-12-03T13:00:33.897752+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/etl_sales.py', '--cfg-path', '/tmp/tmpxngbu1e1']
[2024-12-03T13:00:38.103+0000] {standard_task_runner.py:91} INFO - Job 76: Subtask load_transaction_data
[2024-12-03T13:00:38.146+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_process_dag.load_transaction_data manual__2024-12-03T13:00:33.897752+00:00 [running]> on host a6f8f8b1d92d
[2024-12-03T13:00:38.402+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='k.norasit' AIRFLOW_CTX_DAG_ID='etl_process_dag' AIRFLOW_CTX_TASK_ID='load_transaction_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-03T13:00:33.897752+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-03T13:00:33.897752+00:00'
[2024-12-03T13:00:38.403+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-03T13:00:38.414+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-03T13:00:38.415+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/etl_sales.py", line 49, in load_transaction_data_to_postgres
    objects = minio_client.list_objects_v2(Bucket=bucket_name, Prefix=prefix)
NameError: name 'minio_client' is not defined
[2024-12-03T13:00:38.422+0000] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=etl_process_dag, task_id=load_transaction_data, execution_date=20241203T130033, start_date=20241203T130038, end_date=20241203T130038
[2024-12-03T13:00:38.432+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 76 for task load_transaction_data (name 'minio_client' is not defined; 4338)
[2024-12-03T13:00:38.474+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-12-03T13:00:38.486+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.9/site-packages/***/models/baseoperator.py:1296 AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
[2024-12-03T13:00:38.501+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-03T13:00:38.504+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
